lnwallet/channel.go

```diff
@@ -2532,7 +2549,7 @@ func (lc *LightningChannel) ForceClose() (*ForceCloseSummary, error) {
 //
 // TODO(roasbeef): caller should initiate signal to reject all incoming HTLCs,
 // settle any inflight.
-func (lc *LightningChannel) CreateCloseProposal(feeRate uint64) ([]byte, error) {
+func (lc *LightningChannel) CreateCloseProposal(proposedFee uint64) ([]byte, error) {
```

Comment:
Rather than the fee _entirely_, IMO the fee rate instead should be passed. This lets us keep the internal details of calculating the proper fee (based on the base weight, etc), _within_ the state machine entirely.